examples/resources/configuration/gitrepositories.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gitrepositories.serverless.kyma-project.io
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: gitrepositories.serverless.kyma-project.io
+    busola.io/extension: resource
+data:
+  details: |-
+    {
+      "header": [{
+        "formula": "$exists(data) ? data : 'none'",
+        "highlights": {
+          "informative": ["basic", "key", "none"],
+        },
+        "path": "spec.auth.type",
+        "widget": "Badge"
+      }],
+      "body": []
+    }
+  form: |-
```

Comment:
I asked Jakub why we do have jsons in these yamls - this way we don't have a way to validate its syntax. We already forgot about `"` twice. Let's discuss why not yaml all the way. A bonus feature is `#` comments are allowed in yaml